bin/bump-version

```diff
@@ -49,7 +49,7 @@ else
     done
 fi
 
-rm -f src/{clusterd,environmentd,persist-client,stash-debug}/Cargo.toml.bak LICENSE.bak
+rm -f src/{clusterd,environmentd,persist-client,stash-debug,catalog-debug}/Cargo.toml.bak LICENSE.bak
```

Comment:
while you're in here, could you add testdrive to the second list, please? looks like it's missing

Reply:
I opened a separate PR: https://github.com/MaterializeInc/materialize/pull/22963

I just didn't want the commit message to be too confusing.